ui/Screen/SendFunds.svelte

```diff
@@ -0,0 +1,256 @@
+<script>
+  import { getContext } from "svelte";
+  import { fromStore, toStore, amountStore } from "../src/transfer.ts";
+
+  import { Dropdown, ModalLayout, Rad } from "../DesignSystem/Component";
+  import Row from "../DesignSystem/Component/Transaction/Row.svelte";
+  import {
+    Avatar,
+    Button,
+    Icon,
+    Input,
+    Text,
+    Title,
+  } from "../DesignSystem/Primitive";
+
+  const { identity } = getContext("session");
+  const { orgs } = getContext("session");
+
+  let fromAddress;
+  fromStore.subscribe(value => {
```

Comment:
You can actually just delete these and subscribe to the values automatically when you need to access them, using `$`: https://svelte.dev/tutorial/auto-subscriptions

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/712/commits/059b135f36003ee31ca3b3d725e96732a61fe2bc